app/LiveView.hs

```diff
@@ -12,149 +13,171 @@ module LiveView (
     ) where
 
 import           Control.Monad (void)
-
+import           Data.Version (showVersion)
 import           Terminal.Game
 
-runNodeLiveView :: Int -> IO ()
-runNodeLiveView l = void $
-    runGame (MyState (10, 10) Stop False 1)
-            (logicFun l)
-            drawFun
-            gsQuit
-            13
-
-boundaries :: (Coords, Coords)
-boundaries =
-    ( ( 1 :: Row,  1 :: Column)
-    , (25 :: Row, 95 :: Column)
-    )
-
-data MyState = MyState
-    { gsCoord      :: Coords
-    , gsMove       :: Move
-    , gsQuit       :: Bool
-    , gsMemoryPerc :: Int
+import           Ouroboros.Consensus.NodeId
+import           Paths_cardano_node (version)
+import           Topology
+
+runNodeLiveView :: TopologyInfo -> IO ()
+runNodeLiveView topology = void $
+    runGame (initLiveViewState topology)
+            liveViewLogic
+            liveViewDraw
+            lvsQuit
+            (13 :: FPS)
+
+data LiveViewState = LiveViewState
+    { lvsQuit            :: Bool
+    , lvsRelease         :: String
+    , lvsNodeId          :: Int
+    , lvsVersion         :: String
+    , lvsCommit          :: String
+    , lvsUpTime          :: String
+    , lvsBlockHeight     :: Int
+    , lvsBlocksMinted    :: Int
+    , lvsTransactions    :: Int
+    , lvsPeersConnected  :: Int
+    , lvsMaxNetDelay     :: Int
+    , lvsMempool         :: Int
+    , lvsMempoolPerc     :: Int
+    , lvsCPUUsagePerc    :: Int
+    , lvsMemoryUsageCurr :: Int
+    , lvsMemoryUsageMax  :: Int
     } deriving (Show, Eq)
 
-data Move = N | S | E | W | Stop
-          deriving (Show, Eq)
+initLiveViewState
+    :: TopologyInfo
+    -> LiveViewState
+initLiveViewState (TopologyInfo nodeId _) = LiveViewState
+    { lvsQuit            = False
+    , lvsRelease         = "Shelley"   -- Should be taken from ..?
+    , lvsNodeId          = nodeIdNum
+    , lvsVersion         = showVersion version
+    , lvsCommit          = "bc39f7124" -- Should be taken from ..?
+    , lvsUpTime          = "00:00:00"
+    , lvsBlockHeight     = 0
+    , lvsBlocksMinted    = 0
+    , lvsTransactions    = 0
+    , lvsPeersConnected  = 0
+    , lvsMaxNetDelay     = 0
+    , lvsMempool         = 0
+    , lvsMempoolPerc     = 0
+    , lvsCPUUsagePerc    = 0
+    , lvsMemoryUsageCurr = 0
+    , lvsMemoryUsageMax  = 0
+    }
+  where
+    nodeIdNum = case nodeId of
+        CoreId num  -> num
```

Comment:
could we make this a `nodeId :: Text`? and display for example `C3` or `R1` for core, relay, respectively.